src/main/java/de/hpi/bpt/argos/api/eventQuery/EventQueryEndpointImpl.java

```diff
@@ -163,6 +163,8 @@ public String editEventQuery(Request request, Response response) {
 
         long oldEventQueryId = getEventQueryId(request);
         EventQuery oldEventQuery = PersistenceAdapterImpl.getInstance().getEventQuery(oldEventQueryId);
+				String oldEventQueryString = oldEventQuery.getQuery();
+
         if (oldEventQuery == null) {

```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Change this condition so that it does not always evaluate to "false" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS2583)